core_lwt.opam

```diff
@@ -7,27 +7,16 @@ homepage: "https://github.com/BinaryAnalysisPlatform/core-lwt/"
 bug-reports: "https://github.com/BinaryAnalysisPlatform/core-lwt/issues"
 dev-repo: "git://github.com/BinaryAnalysisPlatform/core-lwt/"
 license: "MIT"
-build: [
-  ["oasis" "setup"]
-  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
-  ["ocaml" "setup.ml" "-build"]
-]
-
-install: [
-  ["ocaml" "setup.ml" "-install"]
-]
-
-remove: [
-  ["ocamlfind" "remove" "core_lwt"]
-]
+build: ["dune" "build" "-p" name "-j" jobs]
```

Comment:
and we need to rename this file as well - dune looks for `<package>.opam`. so we need `core-lwt.opam`